core/builder/src/main/java/io/quarkus/builder/BuildChainBuilder.java

```diff
@@ -1,27 +1,17 @@
 package io.quarkus.builder;
 
+import io.quarkus.builder.item.BuildItem;
+import org.wildfly.common.Assert;
+
 import java.io.BufferedWriter;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStreamWriter;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.ServiceLoader;
-import java.util.Set;
+import java.util.*;
```

Comment:
please avoid using .*

if you undo this the diff will be much cleaner.

Reply:
im sorry i have not done the manual stuff yet. If we merge this, this should be done by computer:

https://github.com/quarkusio/quarkus/pull/45372

Reply:
@maxandersen if you agree with this pr i will undo this change to make it mergeable. If not we can close it. Thanks